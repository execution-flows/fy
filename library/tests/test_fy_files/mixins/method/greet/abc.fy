method greet(greeting: str) -> None
