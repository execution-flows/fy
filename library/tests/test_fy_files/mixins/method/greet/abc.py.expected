import abc


class Greet_MethodMixin_ABC(abc.ABC):
    @abc.abstractmethod
    def _greet(self, greeting: str) -> str | list[int] | None:
        raise NotImplementedError()
